eox_tagging/api/v1/filters.py

```diff
@@ -11,26 +11,26 @@
 class TagFilter(filters.FilterSet):
     """Filter class for tags."""
 
-    course_id = filters.CharFilter(field_name="course_id", method="filter_by_target_object")
-    username = filters.CharFilter(field_name="username", method="filter_by_target_object")
-    enrolled = filters.CharFilter(field_name="enrollment", method="filter_by_target_object")
+    course_id = filters.CharFilter(method="filter_by_target_object")
+    username = filters.CharFilter(method="filter_by_target_object")
+    enrolled_in = filters.CharFilter(method="filter_by_target_object")
```

Comment:
Does this change the external calls? or the filter is defined in the exact same way?

Reply:
Yes. Now if you want to use this instead of doing `enrollment=<COURSE_ID>` is `enrolled_in=<COURSE_ID>`. 

is this change too problematic? I could change `enrolled_in` for `enrollment` (so it works as it did before)

Reply:
Since we have already published the current version yes the change is problematic. Please leave as it was before